src/smart-components/request/action-modal.js

```diff
@@ -1,29 +1,31 @@
-import React, { useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
 import FormRenderer from '../common/form-renderer';
 import { withRouter } from 'react-router-dom';
 import { connect } from 'react-redux';
 import { bindActionCreators } from 'redux';
 import { Modal } from '@patternfly/react-core';
 import { addNotification } from '@red-hat-insights/insights-frontend-components/components/Notifications';
-import { createStageAction } from '../../redux/actions/request-actions';
+import { createStageAction, fetchRequest } from '../../redux/actions/request-actions';
 import { createRequestCommentSchema } from '../../forms/request-comment-form.schema';
 
 const ActionModal = ({
   history: { push },
+  match: { params: { id }},
   actionType,
   addNotification,
   createStageAction,
-  selectedRequest,
   closeUrl,
-  preMethod,
-  postMethod,
-  requestId
+  postMethod
 }) => {
+  const [ selectedRequest, setSelectedRequest ] = useState({});
+
+  const fetchData = (setRequestData) => {
+    fetchRequest(id).payload.then((data) => { setRequestData(data);});
```

Comment:
no need fo brackets around single line functions

```javascript
const fetchData = setRequestData =>
  fetchRequest(id).payload.then(data => setRequestData(data));
```

Reply:
Sounds good. ( had a debug statement that I removed and did not also remove the brackets).